cmd/operator/main.go

```diff
@@ -95,40 +103,51 @@ func main() {
 
 	withWebhook := true
 
-	// TODO(mythi): add a StringVar flag to select which controllers to enable
-	for _, device := range []string{"dsa", "fpga", "gpu", "qat", "sgx"} {
+	switch deviceType {
+	case "fpga":
+		plugins = []string{deviceTypeFpga}
+	default:
+	}
+
+	for _, device := range plugins {
 		if err = setupControllerAndWebhook[device](mgr, ns, withWebhook); err != nil {
```

Comment:
This looks like we're calling setupControllerAndWebhook for all plugins. Should we do it only for chosen plugins?